app/routes/traffic-measure-concepts/index.js

```diff
@@ -25,6 +25,33 @@ export default class TrafficMeasureConceptsIndexRoute extends Route {
       query['filter[label]'] = params.code;
     }
 
-    return await this.store.query('traffic-measure-concept', query);
+    const result = await this.store.query('traffic-measure-concept', query);
+
+    for (let i = 0; i < result.length; i++) {
```

Comment:
nitpicking, if you don't need the index, I prefer the following:
```suggestion
    for (const measure of result) {
```

Reply:
I dont think that works for ember data arrays/collection whatever they are